pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -734,6 +734,8 @@ func (c *Controller) AppendInstanceHandler(f func(*model.ServiceInstance, model.
 			return nil
 		}
 		c.updateEDS(ep)
+
+		log.Debugf("Handle endpoint %s in namespace %s", ep.Name, ep.Namespace)
```

Comment:
In `c.updateEDS` there is already a log of this.

Reply:
remove that looks good

Reply:
Ah, it moved. Ok.